packages/jsii-reflect/lib/property.ts

```diff
@@ -83,4 +83,18 @@ export class Property implements Documentable, Overridable, TypeMember {
   public get docs(): Docs {
     return new Docs(this.system, this, this.spec.docs || {}, this.parentType.docs);
   }
+
+  /**
+   * Return the location in the module
+   */
+  public get locationInModule(): SourceLocation | undefined {
+    return this.spec.locationInModule;
+  }
+
+  /**
+   * Return the location in the repository
+   */
+  public get locationInRepository(): SourceLocation | undefined {
```

Comment:
maybe a base class for Property/Method that reuses docs/location and other stuff

Reply:
Probably a good idea. The next person after me can do it, I already made one for `ClassType/InterfaceType`.